locale/en_US/default.po

```diff
@@ -140,5 +140,7 @@ msgstr "R"
 msgid "default.contextSettings.emailSignature"
 msgstr ""
 "<br/>\n"
-"________________________________________________________________________<br/>\n"
-"<a href=\"{$contextUrl}\">{$contextName}</a>"
+"<hr /><br/>\n"
+"<p class='emaiSignatureContextUrlStyle'><a href=\"{$contextUrl}\">{$contextName}</a></p>"
```

Comment:
Typos: `emai` needs to be `email` throughout

Reply:
Do you have a use in mind for these classes?

Reply:
Fixed the typo.
I added them so that someone could use them in order to style the output using embedded css, but again, that's an email, so you probably shouldn't use css.. I am removing them.